controller/space_iterations.go

```diff
@@ -67,14 +67,13 @@ func (c *SpaceIterationsController) Create(ctx *app.CreateSpaceIterationsContext
 			return jsonapi.JSONErrorResponse(ctx, goa.ErrNotFound(err.Error()))
 		}
 		childPath := append(rootIteration.Path, rootIteration.ID)
-		userActive := false
 		newItr := iteration.Iteration{
 			SpaceID:    ctx.SpaceID,
 			Name:       *reqIter.Attributes.Name,
 			StartAt:    reqIter.Attributes.StartAt,
 			EndAt:      reqIter.Attributes.EndAt,
 			Path:       childPath,
-			UserActive: &userActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c